src/compiler/scala/tools/nsc/Properties.scala

```diff
@@ -13,7 +13,7 @@ object Properties extends scala.util.PropertiesTrait {
 
   // settings based on jar properties, falling back to System prefixed by "scala."
   def residentPromptString = scalaPropOrElse("resident.prompt", "\nnsc> ")
-  def shellPromptString    = scalaPropOrElse("shell.prompt", "\nscala> ")
+  def shellPromptString    = "\n" + Properties.propOrElse("scala.repl.PS1", "scala> ")
```

Comment:
You are right, I was too fast here. 
I propose to keep this one and also try to change namespace as "shell." on other properties in the same way. 
I will rework/rename these and try again (should I go with new PR) ?